stubs/wurlitzer/wurlitzer/__init__.pyi

```diff
@@ -0,0 +1,43 @@
+import contextlib
+import io
+from collections.abc import Iterator
+from contextlib import contextmanager
+from types import TracebackType
+from typing import Any, Final, Literal, TextIO
+from typing_extensions import TypeAlias
+
+STDOUT: Final = 2
+PIPE: Final = 3
+_STDOUT: TypeAlias = Literal[2]
+_PIPE: TypeAlias = Literal[3]
+_InteractiveShell: TypeAlias = Any
```

Comment:
```suggestion

# Alias for IPython.core.interactiveshell.InteractiveShell.
# N.B. Even if we added ipython to the stub-uploader allowlist,
# we wouldn't be able to declare a dependency on ipython here,
# since `wurlitzer` does not declare a dependency on `ipython` at runtime
_InteractiveShell: TypeAlias = Any
```